frontend/src/lib/utils.tsx

```diff
@@ -1138,18 +1139,22 @@ export function endWithPunctation(text?: string | null): string {
     return trimmedText
 }
 
-export function shortTimeZone(timeZone?: string | null, atDate?: Date): string | null {
-    /**
-     * Return the short timezone identifier for a specific timezone (e.g. BST, EST, PDT, UTC+2).
-     * @param timeZone E.g. 'America/New_York'
-     * @param atDate
-     */
-    if (!timeZone) {
+/**
+ * Return the short timezone identifier for a specific timezone (e.g. BST, EST, PDT, UTC+2).
+ * @param timeZone E.g. 'America/New_York'
+ * @param atDate
+ */
+export function shortTimeZone(timeZone?: string, atDate?: Date): string | null {
+    const date = atDate ? new Date(atDate) : new Date()
+    try {
```

Comment:
Additionally, it looks like returning `null` if the provided value is `null` wasn't the right move in #9882. The purpose of `shortTimeZone()` was to return the _user's_ timezone if called without the `timeZone` argument, and that PR changed that. This is now restored.